src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started to work");

```

Comment:
The message should be more informative and indicate that it is a Bulldozer that is working. Consider including the type of machine in the message.